.bacon.yml

```diff
@@ -0,0 +1,50 @@
+test_suites:
+    - name: lint
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '1'
+      timeout: '60'
+      script_name: lint
+      criteria: MERGE
+      queue_name: small
+    - name: unit-karma
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '2'
+      timeout: '10'
```

Comment:
Looking at previous `main` merges, I _think_ this value for our `unit-*` tests will be OK. We may want to bump it to **15** just to be safe.